README.md

```diff
@@ -29,12 +29,14 @@ This repo contains the Claim Status Tracker app, which helps Californians better
   - URL_PREFIX_UIO_DESKTOP
   - URL_PREFIX_UIO_MOBILE
   - URL_PREFIX_BPO
+- (Optional) ASSET_PREFIX: The static asset path (default is `/claimstatus`)
```

Comment:
@lomky I think I would prefer to keep the `(Optional)` on this line because in this list, the items that don't have `(Optional)` are required, meaning that you must specify the env var otherwise the site will 500. You don't need to set `ASSET_PREFIX` because it has a default. Thoughts?

Reply:
but it isnt optional, locally? it errors if it isn't set, which I thought was intended based on the test instructions.

>  Locally, not setting an ASSET_PREFIX env var should not work

Reply:
😅 Good call. I'll tweak the language a little and make it more clear.